tests/test_metrics.py

```diff
@@ -164,3 +164,44 @@ def test_mean_dice():
         results, label, num_classes, ignore_index=255, nan_to_num=-1)
     assert acc[-1] == -1
     assert iou[-1] == -1
+
+
+def test_filename_inputs():
+    import cv2
+    import os
+    import shutil
+
+    def save_arr(input_arrays: list, title: str, is_image: bool):
+        filenames = []
+        PREFIX = './temp'
+        SUFFIX = '.png' if is_image else '.npy'
+        if not os.path.exists(PREFIX):
+            os.makedirs(PREFIX)
```

Comment:
We may use python tempfile library https://docs.python.org/3/library/tempfile.html